### ISSUE TITLE
A.06.03.QoS.availability

### ISSUE THREAD

**Issue by jensscheerlinck** [inspire-eu-validation/ats-discovery-service#59]
_This issue has been extracted from the issue list on:https://ies-svn.jrc.ec.europa.eu/issues/2685_
# Comment

This is not a test for the test framework. Continuous monitoring should be part of the local infrastructure (this applies in general for the other QoS aspects as well, but for them a one-time test in a test run are possible, but not for availability).
# Proposed Change

Remove test case from ATS

**Comment by PeterParslow** [comment]
I disagree with the proposed change. The ATS should include QoS aspects, but it should be clear that these are to be tested locally to the server (described as option 1 at 5.1 of TG Discovery Services). I believe this part of the ATS does not therefore form part of the requirements for the central validation service (that would be option 2 in the TG, which was to be revisited by the IOC TF).
Proposed resolution: flag this test case as 'not for central implementation'
